Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -123,32 +125,22 @@ export class AppComponent implements OnInit, OnDestroy {
       runtimeEnvironment.ravenURL != null &&
       !this.skippedDialogs
     ) {
-      const matDialogRef = this.matDialog.open(BugReportConsentComponent, {
-        hasBackdrop: true,
-      });
-      matDialogRef.afterClosed().subscribe((consent) => {
-        this.store.dispatch(new SetSendFeedback(consent));
-      });
+      this.dialogService.displayDialog('bug-report-consent');
```

Comment:
we should either use enums or modify it to directly put the component into the function as before wit the added benefits of sensible defaults and standard behaviour.

Reply:
magic strings are potential bug source